zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -2460,6 +2498,51 @@ components:
           description: The key of the first process instance the message correlated with
           type: integer
           format: int64
+    MessagePublicationRequest:
+      type: object
+      properties:
+        name:
+          description: The name of the message.
+          type: string
+        correlationKey:
+          description: The correlation key of the message.
+          type: string
+          default: ""
+        timeToLive:
+          description: Timespan (in ms) to buffer the message on the broker.
+          type: integer
+          format: int64
+        messageId:
+          description: |
+            The unique ID of the message. Only useful to ensure only one message with the given ID
+            will ever be published (during its lifetime).
+          type: string
+          nullable: true
+        variables:
+          description: The message variables as JSON document.
+          additionalProperties: true
+          type: object
+          nullable: true
+        tenantId:
+          description: The tenant of the message sender.
+          type: string
+          nullable: true
+      required:
+        - name
+        - correlationKey
+        - timeToLive
```

Comment:
❓ Is this really required? If this is not provided in gRPC, it is `0` by default, simply leading to the message not being buffered on the broker. That should be fine, right?

Reply:
We can argue here, we can also remove `timeToLive` from the required field add the default and minimum value (I don't think a negative value can be provided here)

Reply:
A negative value can actually be provided, the backend is resilient enough and does the same thing for any value `<=0` 🤷

Reply:
Removing it from the required fields allows for easier calls for users since it's actually not required for us, it can be left out and has a proper default behavior then.

Reply:
I'd say, let's remove it from the required field, and add the default value to 0, wdyt ? @tmetzke

Reply:
Sounds good to me 👍